emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBConstants.java

```diff
@@ -62,6 +62,9 @@
   String MAX_BATCH_SIZE = "dynamodb.max.batch.size";
   String MAX_ITEMS_PER_BATCH = "dynamodb.max.batch.items";
 
+  String DELETION_MODE = "dynamodb.deletion.mode";
+  boolean DEFAULT_DELETION_MODE = false;
+
```

Comment:
final?

Reply:
I didn't add it for 2 reasons
1. consistency (nothing in this file uses final)
2. this is an interface method and does not require final